<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MerchantUI</class>
 <widget class="QMainWindow" name="MerchantUI">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1064</width>
    <height>800</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow#MerchantUI{
	
	
	background-image: url(:/image/climain.jpg);
	
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>130</y>
      <width>1061</width>
      <height>681</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget_4{
	
	
	background-image: url(:/image/timg.jpg);
}</string>
    </property>
    <widget class="QWidget" name="widget" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>1051</width>
       <height>551</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widget{
	
	background-color: rgb(255, 255, 255);
	
	
}</string>
     </property>
     <widget class="QLabel" name="lb_head">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>220</y>
        <width>510</width>
        <height>180</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_client">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>530</y>
        <width>76</width>
        <height>21</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#pb_client{
	
	background-image: url(:/ico/pb1.jpg);
}
QPushButton#pb_client{
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 5em;
    padding: 6px;
}
QPushButton#pb_client:hover
{
	
	background-image: url(:/ico/pb.jpg);
    border-style: inset;
}

QPushButton#pb_client:pressed {
	
	background-image: url(:/ico/pb2.jpg);
    border-style: inset;
}


</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>pb_client</zorder>
     <zorder>lb_head</zorder>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>1051</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QPushButton" name="pb_first">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#pb_first{
	background-image: url(:/ico/pb1.jpg);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 5em;
    padding: 6px;
}
QPushButton#pb_first:hover
{
	
	background-image: url(:/ico/pb.jpg);
    border-style: inset;
}

QPushButton#pb_first:pressed{
	
	background-image: url(:/ico/pb2.jpg);
    border-style: inset;
}


</string>
        </property>
        <property name="text">
         <string>用户首页</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pb_mydata">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#pb_mydata{
	
	background-image: url(:/ico/pb1.jpg);
}
QPushButton#pb_mydata{
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 5em;
    padding: 6px;
}
QPushButton#pb_mydata:hover
{
	
	background-image: url(:/ico/pb.jpg);
    border-style: inset;
}



QPushButton#pb_mydata:pressed {
	
	background-image: url(:/ico/pb2.jpg);
    border-style: inset;
}
</string>
        </property>
        <property name="text">
         <string>我的资料</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pb_commodity">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#pb_commodity{
	
	background-image: url(:/ico/pb1.jpg);
}
QPushButton#pb_commodity{
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 5em;
    padding: 6px;
}
QPushButton#pb_commodity:hover
{
	
	background-image: url(:/ico/pb.jpg);
    border-style: inset;
}

QPushButton#pb_commodity:pressed {
	
	background-image: url(:/ico/pb2.jpg);
    border-style: inset;
}


</string>
        </property>
        <property name="text">
         <string>商品管理</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pb_shop">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#pb_shop{
	
	background-image: url(:/ico/pb1.jpg);
}
QPushButton#pb_shop{
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 5em;
    padding: 6px;
}
QPushButton#pb_shop:hover
{
	
	background-image: url(:/ico/pb.jpg);
    border-style: inset;
}

QPushButton#pb_shop:pressed {
	
	background-image: url(:/ico/pb2.jpg);
    border-style: inset;
}


</string>
        </property>
        <property name="text">
         <string>商店管理</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="pb_form">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#pb_form{
	
	background-image: url(:/ico/pb1.jpg);
}
QPushButton#pb_form{
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 5em;
    padding: 6px;
}
QPushButton#pb_form:hover
{
	
	background-image: url(:/ico/pb.jpg);
    border-style: inset;
}

QPushButton#pb_form:pressed {
	
	background-image: url(:/ico/pb2.jpg);
    border-style: inset;
}


</string>
        </property>
        <property name="text">
         <string>订单管理</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="pb_pswd">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#pb_pswd{
	
	background-image: url(:/ico/pb1.jpg);
}
QPushButton#pb_pswd{
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 5em;
    padding: 6px;
}
QPushButton#pb_pswd:hover
{
	
	background-image: url(:/ico/pb.jpg);
    border-style: inset;
}

QPushButton#pb_pswd:pressed {
	
	background-image: url(:/ico/pb2.jpg);
    border-style: inset;
}


</string>
        </property>
        <property name="text">
         <string>修改密码</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>1200</x>
       <y>40</y>
       <width>20</width>
       <height>41</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>校园快跑外卖订餐系统（商家端）</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>当前时间:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lb_time">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>70</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文彩云</family>
      <pointsize>20</pointsize>
      <strikeout>true</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>20</y>
      <width>281</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="1">
      <widget class="QPushButton" name="pb_go">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-image: url(:/ico/pb1.jpg);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 5em;
    padding: 6px;
}
QPushButton:hover
{
	
	background-image: url(:/ico/pb.jpg);
    border-style: inset;
}

QPushButton:pressed{
	
	background-image: url(:/ico/pb2.jpg);
    border-style: inset;
}


</string>
       </property>
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pb_f">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-image: url(:/ico/pb1.jpg);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 5em;
    padding: 6px;
}
QPushButton:hover
{
	
	background-image: url(:/ico/pb.jpg);
    border-style: inset;
}

QPushButton:pressed{
	
	background-image: url(:/ico/pb2.jpg);
    border-style: inset;
}


</string>
       </property>
       <property name="text">
        <string>首页</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>widget_4</zorder>
   <zorder>label_2</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>label_3</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>75</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFirst"/>
   <addaction name="separator"/>
   <addaction name="actionShopinfo"/>
   <addaction name="separator"/>
   <addaction name="actionCommodity"/>
   <addaction name="separator"/>
   <addaction name="actionForminfo"/>
   <addaction name="separator"/>
   <addaction name="actionTalk"/>
   <addaction name="separator"/>
   <addaction name="actionMydata"/>
   <addaction name="separator"/>
   <addaction name="actionUpdatepswd"/>
  </widget>
  <action name="actionFirst">
   <property name="text">
    <string>first</string>
   </property>
  </action>
  <action name="actionMydata">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>mydata</string>
   </property>
  </action>
  <action name="actionShopinfo">
   <property name="text">
    <string>shopinfo</string>
   </property>
  </action>
  <action name="actionCommodity">
   <property name="text">
    <string>commodity</string>
   </property>
  </action>
  <action name="actionForminfo">
   <property name="text">
    <string>forminfo</string>
   </property>
  </action>
  <action name="actionUpdatepswd">
   <property name="text">
    <string>updatepswd</string>
   </property>
  </action>
  <action name="actionTalk">
   <property name="text">
    <string>talk</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
